drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/DMNTypeServiceImpl.java

```diff
@@ -71,7 +71,6 @@ public FactModelTuple retrieveFactModelTuple(Path path, String dmnPath) {
         for (DecisionNode decision : dmnModel.getDecisions()) {
             DMNType type = decision.getResultType();
             checkTypeSupport(type, errorHolder, decision.getName());
-            visitCompositeType(type, false, errorHolder, decision.getName());
```

Comment:
@yesamer could you please explain why we remove this `visitCompositeType`? I thought in my review we just discuss `HashSet` vs `TreeSet`